README.md

```diff
@@ -90,6 +90,12 @@ to run pre-commit directly, you can use
 pre-commit run --all-files
 ```
 
+Use Learning Capabilities
+---------------------------------
+If you want to use the reinforcement learning strategies of Assume please make sure to install torch on top of the requirements listed in the requirements.txt. If you Ressources do not have a GPU available use the CPU only version of torch, which can be installed like this:
+
+pip install torch==2.0.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
+
```

Comment:
Actually for me it just works fine having `pip install torch` - which install torch=2.0.1 from pypi without problems.
So it would be good to double check in a new env if this might be enough..

Then it would be just `pip install -e .[learning]` to install assume with learning dependencies

Reply:
On windows it definetly did not work for me with just the pip install. But good bpoint let's see.

Reply:
For me it did work on windows :-)